technical-writing-project-ideas.md

```diff
@@ -1,22 +1,57 @@
+**Interesting fact:** We call ourselves Content Strategist because user documentation is only one part of our work scope. If you want to know more, let's get in touch at [gsod-tidb@pingcap.com](mailto:gsod-tidb@pingcap.com).
 
-2. Review GitHub Pull Requests about TiDB documentation. This can include reviewing the language and content organization in docs Pull Requests filed by a contributor, a developer, a Database Administrator, or another Technical Writer.
+## Organization overview
 
-3. Execute user acceptance tests for TiDB documentation. This can include executing a usability test by following the existing document, such as testing the deployment of TiDB or TiDB ecosystem tools. The tester might find it necessary to update some steps or outdated description or the content structure, and then the documentation can be improved based on this.
+This section describes the basic information about our open-source organization:
 
-4. Take part in refactoring the TiDB documentation for content reuse of Markdown files. Some existing documents that are a bit too long will be split into two or multiple separate documents, to make it easy for content reuse.
+- **Organization name**: PingCAP
 
-5. (Optional) Improve the [Chinese version of TiDB documentation](https://github.com/pingcap/docs-cn) to make it more user-friendly if the Technical Writer is good at both English and Chinese. This can include reorganizing the content in a single document or within several documents, to make it easier to use and understand.
+- **Organization description**: [PingCAP Inc.](https://pingcap.com/about/) is a software company founded in April 2015, dedicated to building [TiDB](https://github.com/pingcap/tidb), an open-source, distributed, Hybrid Transactional and Analytical Processing (HTAP) database. PingCAP is the primary developer, maintainer, and driver of TiDB and its associated [open-source communities](https://github.com/pingcap/community). It is a venture-backed company, which announced its 50 million USD Series C round financing in September 2018.
+
+- **Reference links:**
+
+    - open-source repository: <https://github.com/pingcap/tidb>
+    - Website: <https://pingcap.com>
+
+- **Email:** [gsod-tidb@pingcap.com](mailto:gsod-tidb@pingcap.com)
+
+- **Display name:**
+
+    - Administrator: [lilin90](https://github.com/lilin90)
+    - Mentor: Coco
+
+## Project ideas
+
+This section describes the project ideas of [TiDB documentation](https://pingcap.com/docs) in Google Season of Docs.
+
+### Project name
+
+TiDB 4.0 Documentation Improvement
+
+### Project description
+
+1. Improve the current [TiDB 4.0 documentation](https://pingcap.com/docs) to make it more user-friendly. This can include reorganizing the content in a single document or within several documents, to make it easier to use and understand.
```

Comment:
```suggestion
1. Improve the current [TiDB 4.0 documentation](https://pingcap.com/docs/v4.0/) to make it more user-friendly. This can include reorganizing the content in a single document or within several documents, to make it easier to use and understand.
```
Should we provide a more specific link?

Reply:
Both are ok for this project.